src/olympia/addons/models.py

```diff
@@ -608,11 +608,6 @@ def automated_signing(self):
         # with on a file-by-file basis.
         return not self.is_listed
 
-    @property
-    def is_sideload(self):
-        # An add-on can side-load if it has been fully reviewed.
```

Comment:
This property was really confusing: it does not indicate that an add-on uses sideloading, but rather than it *can* use sideloading... Happy to see it go.